docs/UserGuide.md

```diff
@@ -687,25 +722,14 @@ Format: `remark_a INDEX [r\REMARK]`
 Note that if the `REMARK` is empty, the existing remarks of the appointment will be deleted.
 </div>
 
-Examples:
+**Examples**:
 * `list_a` followed by `remark_a 2 Patient does not need any follow up` adds the remark `Patient does not need any follow up` to the 2nd appointment in the appointment list.
```

Comment:
missing a r\ in the example. Using this as a reminder only. Will open issue to check all examples are working

Reply:
will fix separately